tasks/holesky/001-pectra-defense/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,114 @@
+
+    function setUp() public {
+        addAllowedStorageAccess(address(OP_DGF));
+        beforeParams[OP_DGF][GameTypes.PERMISSIONED_CANNON] =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.PERMISSIONED_CANNON))));
+
+        beforeParams[OP_DGF][GameTypes.CANNON] =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.CANNON))));
+    }
+
+    function _postCheck(Vm.AccountAccess[] memory accesses, Simulation.Payload memory) internal view override {
+        console.log("Running post-deploy assertions");
+
+        // Does not work on Holesky because the addresses are not in the registry
+        // accesses.decodeAndPrint();
+
+        checkStateDiff(accesses);
+
+        // get the before and after game params for the permissioned game
+        IFaultDisputeGame.GameConstructorParams memory beforeParams_ =
+            beforeParams[OP_DGF][GameTypes.PERMISSIONED_CANNON];
+        IFaultDisputeGame.GameConstructorParams memory afterParams =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.PERMISSIONED_CANNON))));
+        // Set the before params to match the after params, since that is the only thing that should have
+        // changed, the two sets of params should now have the same hash.
+        beforeParams_.absolutePrestate = afterParams.absolutePrestate;
+        require(
+            keccak256(abi.encode(beforeParams_)) == keccak256(abi.encode(afterParams)),
+            "Game params changed unexpectedly"
+        );
+
+        // get the before and after game params for the permissionless game
+        beforeParams_ =
+            beforeParams[OP_DGF][GameTypes.CANNON];
+        afterParams =
+            getGameConstructorParams(IFaultDisputeGame(address(OP_DGF.gameImpls(GameTypes.CANNON))));
+        // Set the before params to match the after params, since that is the only thing that should have
+        // changed, the two sets of params should now have the same hash.
+        beforeParams_.absolutePrestate = afterParams.absolutePrestate;
+        require(
+            keccak256(abi.encode(beforeParams_)) == keccak256(abi.encode(afterParams)),
+            "Game params changed unexpectedly"
+        );
+
+        StandardValidatorV180 validator = StandardValidatorV180(0x3C6423Ce73661f734f100A133Fa996b5F07743C8);
+        StandardValidatorV180.InputV180 memory input = StandardValidatorV180.InputV180({
+            proxyAdmin: SUPERCHAIN_PROXY_ADMIN,
+            sysCfg: SYS_CFG,
+            absolutePrestate: 0x03631bf3d25737500a4e483a8fd95656c68a68580d20ba1a5362cd6ba012a435,
+            l2ChainID: 420110003
+        });
+
+        // Failing to call getProxyImplementation() on some contracts
+        // It seems that the StandardValidator incorrectly assumes that  `SuperchainProxyAdmin == OpChainProxyAdmin`.
+        // console.log("Validator result:", validator.validate(input, false));
```

Comment:
What's the error message that's failing? Is there a bug we need to fix?

Reply:
No error message. The call from the OpChain's ProxyAdmin to the SuperchainProxy reverts and the failure bubbles up.

More discussion in [discord](https://discord.com/channels/1244729134312198194/1244736279384686614/1342989794363576330).